pkg/build/pipelines/redis-valkey/subpackage-cli.yaml

```diff
@@ -0,0 +1,17 @@
+name: Create cli subpackage for redis/valkey
+
+inputs:
+  project:
+    description: |
+      The project name (redis or valkey)
+    required: true
+
+pipeline:
+  - runs: |
+      mkdir -p "${{targets.subpkgdir}}"/usr/bin
+      mv "${{targets.destdir}}"/usr/bin/${{inputs.project}}-cli "${{targets.subpkgdir}}"/usr/bin/${{inputs.project}}-cli
+  
+description: ${{inputs.project}}-cli is the command line interface utility to talk with ${{inputs.project}}.
+
+provides:
+  - ${{inputs.project}}-cli=${{package.full-version}}
```

Comment:
I've never seen such syntax before. Is there example build of how this will be used? I.e. draft pr against wolfi with these pipelines added into ./pipelines/ directory to see this in action?

Reply:
Yeah @xnox I made a PR here that will use this pipeline once it is merged: https://github.com/wolfi-dev/os/pull/30469